mempool/policy_test.go

```diff
@@ -466,11 +466,11 @@ func TestCheckTransactionStandard(t *testing.T) {
 		},
 	}
 
-	timeSource := blockchain.NewMedianTime()
+	pastMedianTime := blockchain.NewMedianTime().AdjustedTime()
```

Comment:
This is the the equivalent of `time.Now()`. I'd suggest using that instead because this is the only thing left in the file that depends on `blockchain` now and thus the dependency could be dropped altogether.